tables/et-6dot.utb

```diff
@@ -0,0 +1,225 @@
+# Estonian 6-dot braille based on the official specification document
+#
+# -----------
+#-index-name: Estonian, 6-dot
```

Comment:
Generally we don't mention "6-dot" in the index-name for literary tables. It is assumed.